src/main/java/http_server/DirectoryCreator.java

```diff
@@ -1,5 +1,6 @@
 package http_server;
 
+import file_handler.FileHandler;
```

Comment:
curiously: is it snake case or usually camel case for the names?

Reply:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

from the above link, i think my package should be `filehandler` and `httpserver` without underscores, but underscores are acceptable

Reply:
interesting!